lib/getHashDigest.js

```diff
@@ -36,15 +43,35 @@ function encodeBufferToBase(buffer, base) {
 	return output;
 }
 
+function getBase(digestType) {
```

Comment:
To simpily this could you:

* Add a `encodeTable` argument to `encodeBufferToBase`
* Add a `encodeBufferToDigestType` which calls `encodeBufferToBase` with the correct arguments according to `digestType`.
* Remove emojis from the `baseEncodeTables` (looks weird)